src/components/admonition/admonition.tsx

```diff
@@ -0,0 +1,124 @@
+import { Component, h, Prop } from '@stencil/core';
+import { AdmonitionType } from '@limetech/lime-elements';
+
+/**
+ * Admonitions, also known as call-outs, very useful for including supportive content
```

Comment:
We had a quick discussion during our standup and @vicgeralds suggested that we should perhaps change the name of this component. This idea was supported by many others, including myself. Can't we just call this component `limel-callout`? 

It's more humanly and universally understandable; and honestly, more correct.

Meaning of _admonition_: https://www.ldoceonline.com/dictionary/admonition
Meaning of _callout_: https://www.ldoceonline.com/dictionary/call-out#call-out__4


Any objections @FlippAre ? 😊

Reply:
I like this idea, much easier to understand

Reply:
Nice. I'll change them all then.